superset/db_engine_specs/trino.py

```diff
@@ -45,6 +46,15 @@ class TrinoEngineSpec(PrestoEngineSpec):
     engine_aliases = {"trinonative"}  # Required for backwards compatibility.
     engine_name = "Trino"
 
+    @classmethod
+    def get_dbapi_exception_mapping(cls) -> Dict[Type[Exception], Type[Exception]]:
```

Comment:
Why is this a function instead of attribute? Do we expect the exceptions to map differently at runtime?